packages/manager/modules/hub/src/components/user-panel/shortcuts/shortcut-tile/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+import component from './component';
+
+import './index.scss';
+
+const moduleName = 'ovhManagerHubShortcutTile';
+
+angular
+  .module(moduleName, ['oui'])
```

Comment:
No translations ?

Reply:
You should import `ovh-ui-angular`

Reply:
i'm not using ui-angular in this component